scene/gui/scroll_container.cpp

```diff
@@ -291,25 +296,13 @@ void ScrollContainer::_update_dimensions() {
 			continue;
 		}
 		Size2 minsize = c->get_combined_minimum_size();
-		child_max_size.x = MAX(child_max_size.x, minsize.x);
-		child_max_size.y = MAX(child_max_size.y, minsize.y);
 
 		Rect2 r = Rect2(-Size2(get_h_scroll(), get_v_scroll()), minsize);
-		if (horizontal_scroll_mode == SCROLL_MODE_DISABLED || (!h_scroll->is_visible_in_tree() && c->get_h_size_flags() & SIZE_EXPAND)) {
```

Comment:
When `*_scroll_mode` is `SCROLL_MODE_SHOW_NEVER`, it is always invisible. As a result, in this case, the resizing cannot be performed. 

Didn't notice before due to display issues with `clip_contents` (#63707).